src/fu-engine.c

```diff
@@ -971,12 +971,19 @@ fu_engine_vendor_quirk_release_version (FuEngine *self, AsApp *app)
 	if (as_app_get_kind (app) != AS_APP_KIND_FIRMWARE)
 		return;
 
-	/* any quirks match */
-	quirk = fu_quirks_lookup_by_glob (self->quirks,
-					  FU_QUIRKS_DAEMON_VERSION_FORMAT,
-					  as_app_get_id (app));
-	if (g_strcmp0 (quirk, "none") == 0)
-		flags = AS_VERSION_PARSE_FLAG_NONE;
+	/* fall back to the quirk database until all files have metadata */
+	quirk = fu_quirks_lookup_by_id (self->quirks,
+					"DaemonVersionFormat=Quad",
```

Comment:
I think you should remain consistent in case, i've seen both Quad and quad.

Reply:
Agreed, fixed.